tmt/utils.py

```diff
@@ -39,6 +40,11 @@
 # Config directory
 CONFIG_PATH = '~/.config/tmt'
 
+# Special process return codes
+PROCESS_TIMEOUT = 124
+PROCESS_KILLED = -9
```

Comment:
Is the minus `-` here intentional?

That doesn't make much sense to me. Return codes should never be negative. Is that some Python thing?

_Note that kill `KILL` signal is `9`   ...    the `-`  for `kill` util is just for it to know it's an [arg](https://shapeshed.com/unix-kill/)._

Reply:
Yes, it's intentionally `-9`. That's what the return code is when `process.kill()` method is called. Don't know much about internal details.

Reply:
Now we're using only the `PROCESS_TIMEOUT` return code.